### PR TITLE
[EN-1388] Move consul alerts role to own repository

### PR DESCRIPTION
**Pull request by agirivera**
Story: http://jira.juwai.com/browse/EN-1388
Sub-task: http://jira.juwai.com/browse/EN-1398
- Adds galaxy tags
- Makes `juwai.consul` a non-conditional dependency so directory creation will not fail
- Simplifies licensing info

How to test:
1. `git clone -b EN-1388 https://github.com/agirivera/ansible-role-consul-alerts.git`
2. `cd ansible-role-consul-alerts/tests`
3. `ansible-galaxy install -p roles -r requirements.yml`. _python27_, _consul_ and _supervisor_ roles will fail for the moment, run the following instead:
   
   ```
   git clone -b EN-1388 https://github.com/agirivera/ansible-role-python27.git roles/juwai.python27
   git clone -b EN-1388 https://github.com/agirivera/ansible-role-supervisor.git roles/juwai.supervisor
   git clone -b EN-1388 https://github.com/agirivera/ansible-role-consul.git roles/juwai.consul
   ```
4. `vagrant up`

/cc @juwai/platform @tjoelsson @imlazyone Please review. Thanks.

### CODE REVIEW

**Comment by tjoelsson** [comment]
I think you need to update the readme

**Comment by agirivera** [comment]
@tjoelsson Updated README.md.

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+Port to bind to, default is 8700
+
+### consul_alerts_supervisor_enabled
+
+Use Supervisord for controlling Consul Alerts, default is true
+
+### consul_alerts_version
+
+Version of Consul Alerts to install, default is 0.3.0
+
```
Comment (tjoelsson) [inline README.md:23]: There are more variables in the default file.

**Comment by tjoelsson** [comment]
Tried to test this but there's a problem in the dependency role: https://github.com/juwai/ansible-role-consul/pull/1#issuecomment-212284304

File: README.md (hunk 1 above)
Comment (agirivera) [inline README.md:23]: I followed the pattern that seems to be done on the other roles I've touch, wherein only ones that can be changed are listed. But, yes, I see your point, we said _available_ :smile_cat:

Comment (agirivera) [inline README.md:23]: Added.

**Comment by agirivera** [comment]
@tjoelsson @imlazyone Fixed issue of dependency. Please review. Thanks.

**Comment by tjoelsson** [comment]
👍

**Comment by agirivera** [comment]
@imlazyone Good to go?

**Comment by stuartwan** [comment]
🚀

**Comment by agirivera** [comment]
@imlazyone Requesting write access. Thanks.

**Pull request merged**